climbing-stairs/HodaeSsi.py

```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def climbStairs(self, n: int) -> int:
+        dp = []
+        dp.append(0)
+        dp.append(1)
+        dp.append(2)
```

Comment:
```suggestion
        dp = [0, 1, 2]
```

이게 더 깔끔할 것 같아서 제안드립니다